packages/components/src/InputDate/InputDate.tsx

```diff
@@ -0,0 +1,57 @@
+import { omit } from "lodash";
+import React from "react";
+import { CommonFormFieldProps, FormField, FormFieldProps } from "../FormField";
+import { DatePicker } from "../DatePicker";
+
+interface InputDateProps
+  extends CommonFormFieldProps,
+    Pick<
+      FormFieldProps,
+      | "readonly"
+      | "disabled"
+      | "onEnter"
+      | "onFocus"
+      | "inputRef"
+      | "validations"
+      | "placeholder"
+      | "onChange"
+      | "onBlur"
+    > {
+  value?: Date;
```

Comment:
Is it possible to add a comment here (I think this may be tricky since it's technically just pulling from FormField?) that explains or has examples of the permitted format(s) for Date? When I was playing in the sandbox I tried passing in "2022-12-01" in the `useState` part of the [sandbox](https://github.com/GetJobber/atlantis/pull/783/files#diff-623e96581af55453f297c86f7710c8426724072352239388c4096912d886356dR27) and it said it wasn't an accepted format.

Reply:
![image](https://user-images.githubusercontent.com/39704901/143333238-148ce229-b65a-479e-b071-1eda07a007da.png)

Reply:
(Not required, but might be nice)

Reply:
@chris-at-jobber Do you think that we should account for other formats? We could do some follow up work around that maybe?

Reply:
Also, just wondering if there should be a comment?  Since it's a `Date` type and people can just check that type and know that certain formats are supported 🤔 Up to you!

Reply:
Hey!  Added a comment in the docs there, it's done in this commit: https://github.com/GetJobber/atlantis/pull/783/commits/f3c90067e6b28e02622088a83fd1a19288cfce5f

I think since it's a `Date` object that is expected typescript will complain if someone tries to pass in a string to the `value`, and know that only certain formats are accepted by the `Date` object.